### DISCUSSION TITLE
Nuclei not found after install

### DISCUSSION THREAD

**Discussion by o0Noise0o** [projectdiscovery/nuclei#5926] [Q&A]
Hello all,

I have ran "go install -v github.com/projectdiscovery/nuclei/v3/cmd/nuclei@latest" without any error but when i want to run nuclei it says it cant find it.nuclei                                                                                                                                         
zsh: command not found: nuclei

Go is installed of course yet i am stuck launching the application. What am i missing here ?

**Comment by Legion-101** [comment]
Hello.
Most likely you forgot to register in PATH the path where the libraries are installed `go get`.
